internal/engine/image_build_and_deployer.go

```diff
@@ -102,6 +102,8 @@ func (ibd *ImageBuildAndDeployer) SetInjectSynclet(inject bool) {
 	ibd.injectSynclet = inject
 }
 
+func (ibd *ImageBuildAndDeployer) MethodName() string { return "[Image Build and] k8s Deploy]" }
```

Comment:
ditto here, the non-matching square brackets seem odd

Reply:
When I saw the brackets, I also wasn't sure how to interpret them. My perspective is — when we're telling users about what Tilt is doing under the hood, we don't necessarily need to be terse and technical. We can't control the error messages that we just pass along, but we're can be thoughtful with the "voice" of the app when it's messaging about how Tilt is working (or not). 

So, just spitballing along those lines. Instead of a bracket, how about something like....
```
k8s deploy, possibly including an image build
```
?

Reply:
This has been changed! Lmk if you still have questions or suggestions but it should be much clearer now. Tl/dr: if we're falling back to it from a LiveUpdate, it necessarily includes an image build (b/c we will only try to live update something if it has an image target).